arm-machinelearning/2017-01-01/swagger/webservices.json

```diff
@@ -0,0 +1,1105 @@
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. This response is returned for an update web service operation. The response payload is identical to the response payload that is returned by the GET operation. The response includes the Provisioning State and the Azure-AsyncOperation header. To get the progress of the operation, call GET operation on the URL in Azure-AsyncOperation header field. For more information about Asynchronous Operations, see https://msdn.microsoft.com/en-us/library/mt742920.aspx.",
+            "schema": {
+              "$ref": "#/definitions/WebService"
+            }
+          },
+          "201": {
+            "description": "Created. This response is returned for a create web service operation. The response includes the Provisioning State and the Azure-AsyncOperation header. To get the progress of the operation, call GET operation on the URL in Azure-AsyncOperation header field. For more information about Asynchronous Operations, see https://msdn.microsoft.com/en-us/library/mt742920.aspx.",
+            "schema": {
+              "$ref": "#/definitions/WebService"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "get": {
+        "tags": [
+          "WebServices"
+        ],
+        "operationId": "WebServices_Get",
+        "description": "Gets the Web Service Definition as specified by a subscription, resource group, and name. Note that the storage credentials and web service keys are not returned by this call. To get the web service access keys, call List Keys.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServiceNameParameter"
+          },
+          {
+            "in": "query",
+            "name": "region",
+            "type": "string",
+            "description": "The region for which encrypted credential parameters are valid.",
+            "required": false
+          },        
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.  The response includes the Provisioning State and the Azure-AsyncOperation header. To get the progress of the operation, call GET operation on the URL in Azure-AsyncOperation header field. For more informationFor more information about Asynchronous Operations, see https://msdn.microsoft.com/en-us/library/mt742920.aspx.",
+            "schema": {
+              "$ref": "#/definitions/WebService"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "WebServices"
+        ],
+        "operationId": "WebServices_Patch",
```

Comment:
Offline chatted last week. We treat this as a legacy naming and has been used by many customers. We will keep it as is in the new release.